HelpSource/Classes/LinuxPlatform.schelp

```diff
@@ -0,0 +1,24 @@
+class:: LinuxPlatform
+summary:: Linux platform-specific methods
+categories:: Platform
+
+description::
+This class is available only on Linux, and implements Linux platform-specific methods.
+
+classmethods::
+
+subsection:: Setting or finding a terminal emulator
+method:: runInTerminalCmd
+get or set the preferred command to invoke a terminal emulator when performing link::Classes/String#-runInTerminal::.
+
+If strong::nil:: (which is the default), sclang will search for a supported terminal emulator installed on the system, the first time strong::runInTerminal:: is called (see link::#*findTerminalEmulator::).
```

Comment:
I think it would be better to use `code::runInTerminal::` instead `strong::runInTerminal::`

Reply:
agreed, I'll change also `strong::nil::` then,